audit/projects/k8s-infra-e2e-boskos-001/services/enabled.txt

```diff
@@ -1,6 +1,9 @@
 NAME                              TITLE
 compute.googleapis.com            Compute Engine API
+containerregistry.googleapis.com  Container Registry API
```

Comment:
Is this scripted?  Who will ultimately clean these up?

Reply:
how to cleanup will be followed up in https://github.com/kubernetes/k8s.io/issues/1675

Yes this is scripted: https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/prow/ensure-e2e-projects.sh

This was added via https://github.com/kubernetes/k8s.io/pull/1536